ezfio.py

```diff
@@ -161,11 +164,17 @@ def ParseArgs():
     physDrive = args.physDrive
     utilization = args.utilization
     outputDest = args.outputDest
+    offset = args.offset
     yes = args.yes
     if (utilization < 1) or (utilization > 100):
         print "ERROR:  Utilization must be between 1...100"
         parser.print_help()
         sys.exit(1)
+
+    if (offset < 0) or (offset > 99):
+        print "ERROR:  offset must be between 0...99"
+        parser.print_help()
+        sys.exit(1)
```

Comment:
One additional condition to test here.  Offset + utilization must be <= 100.  If I just specify `--offset 10` on a 100GB drive it will try to access from 10GB...110GB LBA space, I believe.

Reply:
> One additional condition to test here. Offset + utilization must be <= 100. If I just specify `--offset 10` on a 100GB drive it will try to access from 10GB...110GB LBA space, I believe.

Introduced this test as well. 

> Minor comment about parameter checking above.
> 
> If you don't mind me asking, what is the use for offset? All SSDs today have indirection tables so you're basically going to see the same performance at any offset (but performance will be a funciton of the total written space on the drive).

Yes performance will be same at any given offset. But a use case may arise where one needs to run multiple instances on different regions as a test case.